geo-distributed-deployment-topology.md

```diff
@@ -84,9 +84,13 @@ This section describes the key parameter configuration of the TiDB geo-distribut
               value: "sha"
     ```
 
+   > **Note:**
+   >
+   > Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [Placement Rules](/configure-placement-rules.md).
```

Comment:
```suggestion
   > Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [placement rules](/configure-placement-rules.md).
```